test/arbitrum/compounders/curve/CurveCompounderBaseArbitrumTest.sol

```diff
@@ -324,6 +324,7 @@ contract CurveCompounderBaseArbitrumTest is Test, AddressesArbi {
         vm.prank(harvester);
         uint256 _newUnderlying = curveCompounder.harvest(address(harvester), _asset, 0);
 
+        // From Curve dev discord "Arbitrum is probably a bit funkier -- I haven't dove into it, but I think it requires making a cross-chain call to trigger rewards that may be tough to reproduce"
         // assertTrue(IConvexBasicRewards(curveCompounder.crvRewards()).earned(address(curveCompounder)) == 0, "_testHarvest: E3");
```

Comment:
Could you elaborate on this? What are the implications here? Why would we want this to work and what are the consequences now not to have it?

Reply:
As part of the `harvest` function tests, we push the forked blockchain about a month forward (denominated in blocks) to simulate the effects of having pending rewards for a vault.

In the current Curve/Convex Arbitrum setup, it seems that in order to update the _read_ amount of pending rewards, some cross-chain call needs to be made. The actual rewards (CRV tokens) are distributed as expected.

This doesnt effect the contracts, as the `isPendingRewards` is an external method.